endstate_rew/constant.py

```diff
@@ -14,7 +14,10 @@
 kBT = kB * temperature
 
 zinc_systems = [
-    ("ZINC00061095", "CCOc1ccc2nc(/N=C\c3ccccc3O)sc2c1"),
+    (
+        "ZINC00061095",
+        "CCOc1ccc2nc(/N=C\c3ccccc3O)sc2c1",
+    ),  # NOTE: this system has a wrong topology in the psf file
```

Comment:
Should we also add this comment for the two other systems (ZINC00095858, hipen_id: 6, and ZINC00138607, hipen_id: 11)?

Reply:
yes, good point! will add!